components/SiteCard.tsx

```diff
@@ -32,37 +34,60 @@ const SiteCard = React.forwardRef<HTMLAnchorElement, SiteCardProps>(
           }`}
         >
           <div className="flex flex-col flex-1 w-[60%] p-4 pl-5">
-            <h2 className="text-amber text-xl font-semibold">
-              {isLoading ? 'Loading...' : client_site_name ?? 'My Site'}
-            </h2>
-            <div className="flex flex-col mt-2 gap-1">
-              <p className="text-ocf-gray-500 text-xs font-medium">
-                {`Current output: ${
-                  currentOutput != undefined
-                    ? currentOutput.toFixed(2) + ' kW'
-                    : 'loading...'
+            <div className="flex flex-col flex-1 p-4 pl-5">
```

Comment:
Also does removing this div here still make things work? This will revert the card back to the horizontal style.

Reply:
Idt its this—isn't helping on my end